<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraPropertiesDialog</class>
 <widget class="QDialog" name="CameraPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Camera name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sizeLayout">
     <item>
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>Camera size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="heightBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labPropertiesHeader">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Camera settings:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labAperture">
       <property name="text">
        <string>Aperture (f-stop):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbAperture">
       <property name="toolTip">
        <string>Low value -&gt; shallow Depth of field. High value -&gt; Deep Depth of f ield</string>
       </property>
       <property name="currentIndex">
        <number>5</number>
       </property>
       <item>
        <property name="text">
         <string notr="true">1.4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2.8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">5.6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">22</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameBox</tabstop>
  <tabstop>widthBox</tabstop>
  <tabstop>heightBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CameraPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CameraPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
